packages/desktop-client/src/components/budget/rollover/BalancePopoverContent.tsx

```diff
@@ -3,66 +3,49 @@ import React, { useState } from 'react';
 import { rolloverBudget } from 'loot-core/src/client/queries';
 
 import { useSheetValue } from '../../spreadsheet/useSheetValue';
-import { Tooltip } from '../../tooltips';
 
 import { BalanceMenu } from './BalanceMenu';
-import { CoverTooltip } from './CoverTooltip';
-import { TransferTooltip } from './TransferTooltip';
+import { CoverMenu } from './CoverMenu';
+import { TransferMenu } from './TransferMenu';
 
-type BalanceTooltipProps = {
+type BalancePopoverContentProps = {
   categoryId: string;
-  tooltip: { close: () => void };
   month: string;
   onBudgetAction: (month: string, action: string, arg?: unknown) => void;
   onClose?: () => void;
 };
 
-export function BalanceTooltip({
+export function BalancePopoverContent({
```

Comment:
I would suggest `BalanceMovementMenu` here as we could potentially use this out of the context of popovers.

Reply:
Good idea! https://github.com/actualbudget/actual/pull/2724/commits/d0291ac175fbee88627a05f1ad6aebcd5251680f